bindings/pydeck/examples/02 - Scatterplots.ipynb

```diff
@@ -121,7 +121,7 @@
    },
    "outputs": [],
    "source": [
-    "from IPython.core.display import display\n",
```

Comment:
`display` is also exported from the top-level `IPython` 🤷‍♂️

Reply:
I followed the warning to apply the update. I think `display` in `IPython` is the namespace, not the function.